modules/ipi-install-modifying-install-config-for-slaac-dual-stack-network.adoc

```diff
@@ -0,0 +1,61 @@
+// This is included in the following assemblies:
+//
+// ipi-install-configuration-files.adoc
+
+:_content-type: PROCEDURE
+[id='ipi-install-modifying-install-config-for-slaac-dual-stack-network_{context}']
+= Optional: Configuring address generation modes for SLAAC in dual-stack networks
+
+For dual-stack clusters that use Stateless Address AutoConfiguration (SLAAC), you must specify a global value for the `ipv6.addr-gen-mode` network setting. You can set this value by using NMState to configure the ramdisk and the cluster configuration files. If you don't configure a consistent `ipv6.addr-gen-mode` in these locations, IPv6 address mismatches can occur between CSR resources and `BareMetalHost` resources in the cluster. 
+
+.Prerequisites 
+
+* Install the NMState CLI (`nmstate`).
+
+.Procedure
+
+. Create an NMState YAML file:
++
+[source,yaml]
+----
+interfaces:
+- name: eth0 
+  ipv6:
+    addr-gen-mode: <address_mode> <1>
+----
+<1> You must replace `<address_mode>` with the type of address generation mode required for IPv6 addresses in the cluster. Valid values are `eui64`, `stable-privacy`, or `random`.
+
+. Optional: Consider testing the NMState syntax with the `nmstatectl gc` command before including it in the `install-config.yaml` file, because the installer will not check the NMState YAML syntax.
```

Comment:
Maybe?
```suggestion
. Optional: Consider testing the NMState syntax with the `nmstatectl gc` command before including it in the `install-config.yaml` file, because the installer will not check the NMState YAML syntax automatically.
```